qemu/tests/driver_load.py

```diff
@@ -100,9 +109,14 @@ def get_driver_id(cmd, pattern):
         logging.info("Get driver info from guest")
         driver_id = get_driver_id(driver_id_cmd, driver_id_pattern)
 
+        if stop_service_cmd:
+            stop_service(stop_service_cmd)
         error_context.context("Unload the driver", logging.info)
         unload_driver(driver_unload_cmd, driver_id)
         time.sleep(5)
+
+        if stop_service_cmd:
+            stop_service(stop_service_cmd)
```

Comment:
make a mistake, need to restart service after driver load test. 
updated, thanks